File: soma.py
"""
Programa de somar números
Autor: Leonardo Miguel
Data: 20/03/2022
Descrição:
	Apenas um exercício com a intenção de fazer uma soma,
e exibir o resultado para o usuário que estiver utili-
zando este programa.
"""

# Às variáveis (num1 e num2) tem a função de armazenar os digitos pelo o usuário.
num1 = int(input("Digite um número para soma:  "))
num2 = int(input("Digite outro numero para soma: "))    

# A variável (soma) tem o trabalho de fazer de fato a adição das variáveis (num1 e num2).
soma = num1 + num2

    
print(f'A soma dos números é: {soma}')

File: ruby/soma.rb
# Declaração de variáveis 


=begin
    
    Nessa parte há puts para que o usuário identifique que ele
tem que colocar números nos (gets);
    E nos gets de fato são nossas variáveis que irão pegar os 
números que forem digitados pelo o usuário.
    
=end
puts "Insira um número para a soma: "
num1 = gets
puts "Insira outro número: "
num2 = gets


=begin


=end

soma = Integer(num1) + Integer(num2)

puts "A soma dos números é: #{soma}!"